spec/features/user_admin_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "admin can modify and delete users", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.